influxdb3_server/src/http.rs

```diff
@@ -419,6 +419,13 @@ where
                 .await?
         };
 
+        let num_lines = result.line_count;
+        let payload_size = body.len();
+        let telem_store = Arc::clone(&self.common_state.telemetry_store);
+        tokio::spawn(async move {
```

Comment:
I don't think we want to spawn on every write call. We should be able to just call add_write_metrics on the store without this.